controllers/storagecluster/external_resources.go

```diff
@@ -380,6 +381,9 @@ func (r *StorageClusterReconciler) createExternalStorageClusterResources(instanc
 				scc = newCephBlockPoolStorageClassConfiguration(instance)
 				// update the storageclass name to rados storagesclass name
 				scc.storageClass.Name = fmt.Sprintf("%s-%s", instance.Name, d.Name)
+			} else if d.Name == cephRbdTopologyStorageClassName && instance.Spec.ManagedResources.CephNonResilientPools.Enable {
```

Comment:
> instance.Spec.ManagedResources.CephNonResilientPools.Enable

Why we need this setting to get enabled for external mode?

Reply:
Hm, depends on the usage flow you want. If you want the storageclass to be created whenever the json has the details for it we can skip the flag for external mode.

Reply:
Exactly we dont need another option to enable it, its totally on the ceph admin to enable it